bayesian_cnn_prometheus/preprocessing/data_generator.py

```diff
@@ -77,6 +100,22 @@ def _generate_data(self, dataset_type: DatasetType, batch_size: int):
                         images_chunks, targets_chunks = self._shuffle_chunks(images_chunks, targets_chunks)
                     yield np.array(images_chunks), np.array(targets_chunks)
 
+    def _image_flow(self, dataset_type: str):
+        assert self.dataset_structure is not None
```

Comment:
Czy używanie assert poza testami jest ok? :D

Reply:
Ja uważam że tak i że jest dobrą praktyką, dlatego że wtedy możesz mieśc pewność odnośnie tego na jakich danych operujesz. 
A oprócz tego możesz zwrócić czytelny komunikat do programisty co do tego co jest nie tak.

Reply:
A nie powinno się wtedy robić try except?

Reply:
Zależy od tego czy tworzysz narzędzie, czy bibliotekę. 
Jeśli tworzy bibliotekę, to według mnie nie. Powinieneś dać jasny feedback programiści.
Jeśli tworszysz narzędzie to tak, bo słabo kiedy użytkownik końcowy widzi jak aplikacja się wywala